src/pages/Landing/index.tsx

```diff
@@ -117,7 +118,7 @@ export default () => {
             onChange={(value) => setNewUser({ ...newUser, isCool: value })}
             isChecked={newUser.isCool}
           />
-          <Button type='submit'>Submit</Button>
+          <SnazzyButton isSnazzy={true} type='submit'>Submit</SnazzyButton>
```

Comment:
#nit
isSnazzy={true} 
Could be: 
isSnazzy

[The default value of unspecified props is true](https://legacy.reactjs.org/docs/jsx-in-depth.html#props-default-to-true)

Reply:
Good to know! 😀